pkg/command/status.go

```diff
@@ -26,6 +26,9 @@ func StatusCommand(ctx context.Context, cfg config.Config, repo *git.Repository)
 	if err != nil {
 		return "", fmt.Errorf("failed getting pr for current branch: %w", err)
 	}
+	if pr.State.GetPRType() == git.PRTypeFeature {
+		return "Automatically allowing feature branch PR", nil
+	}
```

Comment:
Will this check ever trigger? Won't the branch prefix check above catch this case before we get here? I think this is a good message, so it should prolly be moved up before the prefix check?